xml/repeated-content-decl.ent

```diff
@@ -0,0 +1,118 @@
+<!--...........................................................................
+Repeated content entities, for re-printing content in multiple locations
+rather than using xrefs, so that users do not have to skip around so much
+This works just like copy-paste, so be mindful of sections, titles, and
+paragraphs
+............................................................................-->
+
+<!-- Use xmlns="http://docbook.org/ns/docbook" on the first occurrence of every
+formatting tag in each entity, to avoid "Namespace default prefix was not found" 
+errors -->
+
+<!ENTITY namespaces 
+'<formalpara xmlns="http://docbook.org/ns/docbook">
+<title>Length of release names</title> 
+<para xmlns="http://docbook.org/ns/docbook">
+Release names (e.g. when you run <command>helm install --name</command>) 
```

Comment:
Avoid the use of "e.g.", it should be "for example" (see http://docserv.suse.de/documents/Style_Guide/docu_styleguide/html/#sec.latin)